conf/kaust.config

```diff
@@ -4,12 +4,11 @@ params {
     config_profile_description = 'Profile for use on King Abdullah Univesity of Science and Technology (KAUST) Ibex Cluster.'
     config_profile_contact = 'Husen Umer (@kaust.edu.sa)'
     config_profile_url = 'https://docs.hpc.kaust.edu.sa/quickstart/ibex.html'
-    save_reference = false
     igenomes_ignore = true
 }
 
 // Load genome resources and assets hosted by the Bioinformatics team on IBEX cluster
-// includeConfig '/biocorelab/BIX/resources/configs/genomes.yaml'
+includeConfig '/biocorelab/BIX/resources/configs/genomes.config'
```

Comment:
This will not work because it cannot be linted/tested here as the GitHub actions nodes will not have access this file!

Please ask on #configs on slack for suggestions how to get around this

Reply:
I know that is not working and I tried to find some solution through the [configs channel](https://nfcore.slack.com/archives/CGRTMASKY/p1733662786166109) but have not gone much so far. I have reposted a question, will see how it goes. 
The base line is that although it is not working with the profile but when the institute config file implicitly specified with `-c` flag then it works therefore I thought of keeping it here even though the lint test fails.